jinahub/indexers/searcher/compound/FaissPostgresIndexer/README.md

```diff
@@ -1,28 +1,29 @@
 # FaissPostgresIndexer
 
-**FaissPostgresIndexer** is a compound Executor for Jina, made up of [FaissSearcher](https://hub.jina.ai/executor/gilkzt3f) for performing similarity search on the embeddings, and of [PostgreSQLStorage](https://hub.jina.ai/executor/d45rawx6) for retrieving the metadata of the Documents. 
+**FaissPostgresIndexer** is a compound Executor for Jina, made up of [FaissSearcher](https://hub.jina.ai/executor/gilkzt3f) for performing similarity search on the embeddings, and of [PostgreSQLStorage](https://hub.jina.ai/executor/d45rawx6) for retrieving the metadata of the Documents.
 
+## Usage
 
-Additionally, you will need a running PostgreSQL database. This can be a local instance, a Docker image, or a virtual machine in the cloud. Make sure you have the credentials and connection parameters.
+**Note** that you will need a running PostgreSQL database. This can be a local instance, a Docker image, or a virtual machine in the cloud. Make sure you have the credentials and connection parameters.
 
 You can start one in a Docker container, like so: 
 
 ```bash
 docker run -e POSTGRES_PASSWORD=123456  -p 127.0.0.1:5432:5432/tcp postgres:13.2 
 ```
 
-## Usage
+This is a combination of an Indexer and a Searcher. Thus, you can perform all the CRUD operations on it, while still being able to search.
```

Comment:
Add a link to Indexers in docs.jina and make that page list the endpoints for CRUD and search